x-pack/plugins/apm/public/components/app/mobile/transaction_overview/transaction_overview_tabs/stats_list/index.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { ManagedTable } from '../../../../../shared/managed_table';
+import { APIReturnType } from '../../../../../../services/rest/create_call_apm_api';
+import { getColumns } from './get_columns';
+
+type MobileMainStatisticsByField =
+  APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/main_statistics'>['mainStatistics'];
+
+type MobileDetailedStatisticsByField =
+  APIReturnType<'GET /internal/apm/mobile-services/{serviceName}/detailed_statistics'>;
+
+interface Props {
+  isLoading: boolean;
+  mainStatistics: MobileMainStatisticsByField;
+  detailedStatisticsLoading: boolean;
+  detailedStatistics: MobileDetailedStatisticsByField;
+  comparisonEnabled?: boolean;
+  offset?: string;
+}
+export function StatsList({
+  isLoading,
+  mainStatistics,
+  detailedStatisticsLoading,
+  detailedStatistics,
+  comparisonEnabled,
+  offset,
+}: Props) {
+  const columns = useMemo(() => {
+    return getColumns({
+      detailedStatisticsLoading,
+      detailedStatistics,
+      comparisonEnabled,
+      offset,
+    });
+  }, [
+    detailedStatisticsLoading,
+    detailedStatistics,
+    comparisonEnabled,
+    offset,
+  ]);
+  return (
+    <ManagedTable
+      noItemsMessage={
+        isLoading
+          ? i18n.translate('xpack.apm.statsTable.loading', {
+              defaultMessage: 'Loading...',
+            })
+          : i18n.translate('xpack.apm.statsTable.noDataLabel', {
+              defaultMessage: 'No data found',
+            })
+      }
```

Comment:
`statsTable` seems broad, and don't we have labels or a component that we can reuse to wrap it in a loading state? I thought we did but maybe we removed it.

Reply:
I could find a component for that, I checked how we did for other tables
I will improve the i18n label

Reply:
Addressed here https://github.com/elastic/kibana/pull/153044/commits/a79ebaa66ebd32c6ef351b2b7eaddfc3acd60c6c